### PR TITLE
chore: replace RTB <Input>, final RTB cleanup

### PR DESCRIPTION
**Pull request by suprko**
#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/1340/desktop-replace-input-components
https://app.clubhouse.io/peerio/story/2834/peerui-final-cleanup 

#### Testing instructions  
1. Changes to `package.json` -> verify that it actually works for you! I rm'd all node_modules and installed again, works for me, but plz check on your side too :)
2. Then can test on `dev` as with other RTB replacements

----
### Repository owner

- [ ] Was tested and can be merged.
- [ ] PR name follows conventional changelog format.

[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)

### CODE REVIEW

File: src/ui/chat/ChatView.jsx [hunk 1]
```diff
@@ -113,10 +113,6 @@ class ChatView extends React.Component {
         this.chatNameEditorVisible = false;
     };
 
-    chatNameEditorRef = ref => {
-        if (ref) ref.nameInput.focus();
-    };
-
     postJitsiLink = () => {
```
Comment (seavan) [inline src/ui/chat/ChatView.jsx]: use @action.bound

File: src/ui/chat/components/sidebar/ChannelSideBar.jsx [hunk 2]
```diff
@@ -66,6 +69,22 @@ class ChannelSideBar extends React.Component {
         }
     }
 
+    handleKeyDown = () => {
```
Comment (seavan) [inline src/ui/chat/components/sidebar/ChannelSideBar.jsx]: use @action.bound

File: src/ui/chat/components/sidebar/ChannelSideBar.jsx [hunk 3]
```diff
@@ -66,6 +69,22 @@ class ChannelSideBar extends React.Component {
...
+        this.resizeTextarea();
+    }
+
+    resizeTextarea = () => {
```
Comment (seavan) [inline src/ui/chat/components/sidebar/ChannelSideBar.jsx]: use @action.bound

**Comment by anri-asaturov** [comment]
resolve conflicts plz

**Comment by suprko** [comment]
@anri-asaturov done

**Pull request merged**